server/database/database.js

```diff
@@ -14,9 +14,22 @@ con.connect( err => {
         console.error(err);
     }
     console.log('Connected to RDS MySQL database!');
-})
+});
+
+const userNames_passwords = (data, callback) => {
+    const queryString = "SELECT userNames, passwords FROM Users";
+    con.query(queryString, (err,data) => {
+        if (err) {
+            console.log(err, "Error getting userNames and passwords");
+        };
+        callback(null, data);
+    });
+};
+
+
 
 
 
 
-module.exports = {con};
\ No newline at end of file
+module.exports = {con};
+module.export.userNames_passwords = userNames_passwords;
```

Comment:
Remove this line. On line 34, after con, add a comma and type the function name, so it should look like, module.exports = {con, <function name>};